cmd/flipt/evaluate.go

```diff
@@ -137,12 +137,33 @@ func (c *evaluateCommand) run(cmd *cobra.Command, args []string) error {
 		case <-ticker.C:
 			err := c.evaluate(ctx, flag.Type, sdk, request)
 			if err != nil {
-				defaultLogger.Error("failed to evaluate", zap.Error(err))
+				fmt.Printf("failed to evaluate: %s", err)
```

Comment:
```suggestion
				fmt.Printf("failed to evaluate: %w", err)
```
The common pattern for error output in the project

Reply:
i dont think it'll make any different in output as `%w` is used to wrap errors when returning a new error. In our case we're just printing to stdout so %w would have no real effect

Reply:
make sense